### PR TITLE
Beta 4 release

### PR DESCRIPTION
**Pull request by clowerweb**

### CODE REVIEW

File: webpack-bundle-report.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
```
Comment (johnnynotsolucky) [inline webpack-bundle-report.html:3]: This file can be deleted and untracked.

File: src/resources/components/Chart.vue [hunk 2]
```diff
@@ -1,654 +1,660 @@
...
-							point: {
-								radius: 0,
-							},
-						},
-						tooltips: {enabled: false},
-						scales: {
-							y: {
-                  beginAtZero: true,
-									display: false,
+import Chart from 'chart.js/auto';
```
Comment (johnnynotsolucky) [inline src/resources/components/Chart.vue:2]: Going forward please make formatting changes in a separate commit (same branch/PR is fine though), it's pretty hard to work out what has changed.

**Review by johnnynotsolucky — CHANGES_REQUESTED** [review]
@clowerweb where is the change which reuses the query to fetch the data for charts and graphs?

Please update CHANGELOG.md with changes.

**Comment by clowerweb** [comment]
> @clowerweb where is the change which reuses the query to fetch the data for charts and graphs?
> 
> Please update CHANGELOG.md with changes.

No change was necessary; it already worked that way. I'm unsure how the conclusion was met that it didn't and something needed changed, but unless I misunderstood the task requirements, it already did what we wanted.

File: src/resources/components/Chart.vue (hunk 2 above)
Comment (clowerweb) [reply src/resources/components/Chart.vue:2]: My IDE does it automatically whenever I save, but I'll turn that feature off.

**Review by johnnynotsolucky — APPROVED** [review]
